cli/module_registry.go

```diff
@@ -43,7 +43,9 @@ type moduleID struct {
 
 // moduleManifest is used to create & parse manifest.json.
 type moduleManifest struct {
-	Name        string            `json:"name"`
+	// for backward compatibility - DO NOT SET as will be deprecated
+	Name        string            `json:"name,omitempty"`
```

Comment:
setting omitempty and now clearing it will mean that it doesn't show up in the json unless its set